comp/volunteers/blue-shirts.yaml

```diff
@@ -1,6 +1,6 @@
-summary: Order additional blueshirts for the competition
+summary: Order volunteer shirts for the competition
```

Comment:
Question: Should designing and ordering blueshirts be the same task? Feels unnecessary to separate designing them as a sub-task of "swag"